functions/graphql.js

```diff
@@ -89,16 +89,16 @@ const localExecutableSchema = makeExecutableSchema({
 
 const {
   overrideTypeDefs,
-  createOverrideResolvers
+  createOverrideResolvers,
 } = require('./graphql/overrideSchema')
 
 // *****************************************************************************
 // 4) put it all together
 // *****************************************************************************
 
 const schema = mergeSchemas({
-  schemas: [overrideTypeDefs, localExecutableSchema, transformedRemoteSchema],
-  resolvers: createOverrideResolvers(remoteExecutableSchema)
+  schemas: [transformedRemoteSchema, overrideTypeDefs, localExecutableSchema],
```

Comment:
The overrideTypeDefs need be merged after the remote schema.

The remote schema (straight from Fauna) sends back the secret as a string, which gets put into the cookie.  The override version returns a Boolean.

After fixing the override login and loggedIn return values to Boolean, I realized calling the login mutation was still returning `"true"` or `"false"` strings.  Swapping these fixed that, and now it correctly returns a Boolean every time.